### PR TITLE
Further edits to 2020 Markup chapter

### PR DESCRIPTION
**Pull request by j9t**
Some small tweaks and improvements based on a recent chapter review.

Progress on #899 #1432

### CODE REVIEW

File: src/content/en/2020/markup.md [hunk 1]
```diff
@@ -983,7 +983,7 @@ The two most popular ones stand out because they are almost twice as popular tha
...
 
 {# TODO(authors): Update this interpretation given that the lazy loading stat is in terms of pages, not img elements. #}
-Interestingly, the use of native lazy loading on images is similar to that of `data-src`. 3.86% of pages use the `<img loading="lazy">` attribute. This appears to be growing very fast, as back in February, this number was about [0.8%](https://twitter.com/zcorpan/status/1237016679667970050). It's possible that these are being used together for a [cross-browser solution](https://addyosmani.com/blog/lazy-loading/).
+Interestingly, the use of native lazy loading on images is similar to that of `data-src`. 3.86% of pages use `loading="lazy"` on `img` elements. This appears to be growing very fast, as back in February, this number was about [0.8%](https://twitter.com/zcorpan/status/1237016679667970050). It's possible that these are being used together for a [cross-browser solution](https://addyosmani.com/blog/lazy-loading/).
```
Comment (j9t) [inline src/content/en/2020/markup.md:986]: @Tiggerito, I couldn’t find any reference for what we looked at in the results spreadsheet. Was this use of `loading` actually limited to `img` elements, or also to `iframe`s? Just to err on the safe side.

Comment (bazzadp) [reply src/content/en/2020/markup.md:986]: I added a link in #1858 as also found it difficult to substantiate that figure. I believe this was reused from @Tiggerito 's SEO analysis: https://docs.google.com/spreadsheets/d/1ram47FshAjzvbQVJbAQPgxZN7PPOPCKIK67VJZCo92c/edit#gid=2109061092 as it has this 3.86% stat.

That uses this query, which is just for images: https://github.com/HTTPArchive/almanac.httparchive.org/blob/main/sql/2020/07_SEO/pages_markup_by_device_and_img_loading.sql

**Review by bazzadp — APPROVED** [review]
Approved but next time could you [reset your `main` branch](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Git-Guide#commands-to-force-reset-your-main-branch-to-upstream) to avoid retagging old commits?

File: src/content/en/2020/markup.md (hunk 1 above)
Comment (Tiggerito) [reply src/content/en/2020/markup.md:986]: Yes, it's just for images. I separately gathered the same data for iframes.

pages_markup_by_device_and_iframe_loading.sql

Comment (j9t) [reply src/content/en/2020/markup.md:986]: Thanks for checking!

**Comment by j9t** [comment]
Thanks @bazzadp, @Tiggerito.

@bazzadp, can you merge this one, or would you suggest additional reviews?

**Pull request merged**